src/ValuedTextInput/valuedTextInput.css

```diff
@@ -117,7 +118,10 @@
 
 .error
 {
-    .valueLabel
+    &:hover .valueLabel,
+    .valueLabel,
+    &:hover .input,
+    .input
```

Comment:
I think this would be more readable if expressed as:
```
.error,
.error:hover
{
    .input,
    .valueLabel
    {
        border-color: var( --AC-2 );
    }
}
```